components/meta-packages/sunray-essential/sunray-essential.p5m

```diff
@@ -0,0 +1,107 @@
+depend fmri=library/tooltalk@0.5.11 type=require
+depend fmri=library/motif@0.5.11 type=require
+
+# gdm-session-worker 32bit libraries
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libglib-2.0.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/mps/libnspr4.so
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/mps/libnss3.so
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libdbus-glib-1.so.2
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libdbus-1.so.3
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libpcre2-8.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libffi.so.6
+
+# utaudio 32bit libs
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libpulse.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libmpg123.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libFLAC.so.14
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libsndfile.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libSDL2-2.0.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libvorbis.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libopus.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libogg.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libmp3lame.so
+
+# uttsc-bin (RDP client)
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libX11.so.4
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXau.so.6
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libxcb.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXdmcp.so.6
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXext.so.0
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXfixes.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXft.so.2
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXcursor.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXrender.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXv.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXinerama.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXrandr.so.2
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libXpm.so.4
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libz.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libpng16.so.16
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libfontconfig.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libfreetype.so.6
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libbrotlidec.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libbz2.so.1
+depend type=require fmri=__TBD pkg.debug.depend.file=usr/lib/libexpat.so.1
+
+# utresexec
```

Comment:
Could you please elaborate what exactly this means?

Maybe these cryptic comments are clear enough for you, but these should be fully understandable for anybody with zero SRSS knowledge.

Reply:
These are libraries which are used by utresexec, an exec wrapper